worldedit_commands/init.lua

```diff
@@ -991,7 +995,10 @@ minetest.register_chatcommand("/restore", {
 minetest.register_chatcommand("/save", {
 	params = "<file>",
 	description = "Save the current WorldEdit region to \"(world folder)/schems/<file>.we\"",
-	privs = {worldedit=true},
+	privs = {
+		worldedit=true,
+		worldedit_global=true,
+	},

```

Comment:
I actually like the multi-line variant since it allows line-based editing. In this case, it may be of limited use, though.